generators/generator-base.js

```diff
@@ -488,9 +489,10 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
      * @param {string} route - The route for the module. For example 'entity-audit'.
      * @param {string} modulePath - The path to the module file. For example './entity-audit/entity-audit.module'.
      * @param {string} moduleName - The name of the module. For example 'EntityAuditModule'.
+     * @param {string} pageTitle - The translation key or the text for the page title in the browser. For example 'entityAudit.home.title' or 'Entity audit'.
      */
-    addAdminRoute(route, modulePath, moduleName) {
-        this.needleApi.clientAngular.addAdminRoute(route, modulePath, moduleName);
+    addAdminRoute(route, modulePath, moduleName, pageTitle) {
```

Comment:
```sugestion
addAdminRoute(route, modulePath, moduleName, pageTitle = moduleName);
```

Reply:
Can't do this way because if i18n is enabled then this is translation key and this shows `translation-not-found` error in page title.
It should be undefined by default as this is now - then general page title is shown, which is better than `translation-not-found`.
Improving parameter comment in coming PR to give that additional information there.

Reply:
`this.enableTranslation ? undefined : moduleName`?

Reply:
I prefer not to add this, `moduleName` can be technical term, so better to show global page title rather than technical term.